### PR TITLE
Added Certificate issuer.

### PR DESCRIPTION
**Pull request by jacobcrawford**
Added the possibility to set up a certificate issuer.

### CODE REVIEW

File: chart/templates/cluster-issuer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{{ range .Values.issuers }}
+{{ if .enabled }}
```
Comment (skovbakke) [inline chart/templates/cluster-issuer.yaml:2]: If the issuer is not enabled it should not be in the values file. The field should also be removed from the values file.

**Review by skovbakke — CHANGES_REQUESTED** [review]
Good besides the one comment

File: chart/templates/cluster-issuer.yaml (hunk 1 above)
Comment (jacobcrawford) [reply chart/templates/cluster-issuer.yaml]: done.

**Review by skovbakke — APPROVED** [review]
Good

**Pull request merged**